common/src/main/resources/messages/messages.properties

```diff
@@ -241,6 +241,10 @@ cluster.upgrade.download.parcel=Downloading the new parcel
 cluster.upgrade.distribute.parcel=Distributing the new parcel
 cluster.upgrade.activate.parcel=Activating the new parcel
 cluster.upgrade.start.upgrade=Calling upgrade on Runtime
+cluster.upgrade.validation.started=Cluster upgrade validation started.
```

Comment:
Please remove the trailing `.` from these messages as we don't use it for other notifications either.